code/modules/projectiles/guns/misc/syringe_gun.dm

```diff
@@ -102,3 +102,41 @@
 		else
 			to_chat(user, "<span class='warning'>[src] cannot hold more syringes!</span>")
 	return FALSE
+
+/obj/item/gun/syringe/chem
+	name = "reagent gun"
+	desc = "A Nanotrasen syringe gun, modified to automatically synthesise chemical darts, and instead hold reagents."
+	icon_state = "chemgun"
+	item_state = "chemgun"
+	max_syringes = 4
+	container_type = OPENCONTAINER
+	var/time_per_syringe = 250
+	var/last_synth = 0
+
+/obj/item/gun/syringe/chem/attackby(obj/item/A, mob/user, params, show_msg = TRUE)
+	return

```

Comment:
Pretty sure @kevinz000 made every attackby not do this

Reply:
attackby here exists to stop people from loading syringes into the gun, since you're not supposed to be able to do so

Reply:
attackby is suppose to send signals to components so long-term this should probably be changed.

Reply:
Still needs changing.